docs/census_notebook_guidelines.md

```diff
@@ -31,19 +32,19 @@ Examples:
 Introductory text must be included right underneath the title.
 
 * It must provide a one paragraph summary of the notebook's goals.
-* It must not contain an explanation of the Cell Census.
+* It must not contain an explanation of the Census.
 
 Examples: 
 
 :white_check_mark:
 
-> This notebook provides a demonstration for integrating two Cell Census datasets using scvi-tools. The goal is not to provide an exhaustive guide on proper integration, but to showcase what information in the Cell Census can inform data integration.
+> This notebook provides a demonstration for integrating two Census datasets using scvi-tools. The goal is not to provide an exhaustive guide on proper integration, but to showcase what information in the Cell Census can inform data integration.
```

Comment:
confusing as it mentions "Census" - how does this tie into the "no direction mention" guideline?

Reply:
isn't that rule just for the title?